src/main/java/org/skriptlang/skript/bukkit/itemcomponents/equippable/elements/ExprSecBlankEquipComp.java

```diff
@@ -0,0 +1,109 @@
+package org.skriptlang.skript.bukkit.itemcomponents.equippable.elements;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.config.SectionNode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Example;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SectionExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.Trigger;
+import ch.njol.skript.lang.TriggerItem;
+import ch.njol.skript.registrations.EventValues;
+import ch.njol.skript.variables.Variables;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.itemcomponents.equippable.EquippableExperiment;
+import org.skriptlang.skript.bukkit.itemcomponents.equippable.EquippableWrapper;
+
+import java.util.List;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+@Name("New Equippable Component")
+@Description("Gets a blank equippable component. "
+	+ "NOTE: Equippable component elements are experimental. Thus, they are subject to change and may not work aas intended.")
+@Example("""
+	set {_component} to a blank equippable component
+	set the equippable component of {_item} to {_component}
+	""")
+@Example("""
+	set {_component} to a blank equippable component:
+		set the equipment slot to chest slot
+	""")
+@RequiredPlugins("Minecraft 1.21.2+")
+@Since("INSERT VERSION")
+public class ExprSecBlankEquipComp extends SectionExpression<EquippableWrapper> implements EquippableExperiment {
+
+	public static class BlankEquippableSectionEvent extends Event {
```

Comment:
```suggestion
	private static class BlankEquippableSectionEvent extends Event {
```
I'd keep access to these classes as limited as possible